modules/ROOT/pages/cloud-troubleshooting.adoc

```diff
@@ -55,31 +56,31 @@ The `+apiKey+` must be:
 
 ==== Solution
 
-Check the `+apiKey+` provided in the script tag:
```

Comment:
I think the previous format of havin one level deep bullet points looks better.
<img width="1550" alt="image" src="https://github.com/tinymce/tinymce-docs/assets/40410898/c681abdc-b584-4282-86cc-0630e005caf2">

Reply:
updated